client/src/tests/CourseView.test.js

```diff
@@ -107,31 +107,4 @@ describe('Tests for CourseView component', () => {
     expect(firstTeacherInCharge).not.toBeInTheDocument();
   });
 
-  test('CourseView should show a dialog for uploading a file when clicking on a menu button and choosing that option', async () => {
```

Comment:
Why is this test deleted?

Reply:
I tink this test was moved to the FileLoadDialog.test file

Reply:
Yep, it was just moved.